rskj-core/src/main/java/co/rsk/core/types/bytes/Bytes.java

```diff
@@ -106,6 +106,14 @@ static boolean equalBytes(Bytes b1, Bytes b2) {
      * @return the wrapped by this instance byte array.
      */
     byte[] asUnsafeByteArray();
+
+    static void arraycopy(byte[] orig, int srcPos, byte[] dest, int destPos, int length) {
```

Comment:
seems this one duplicates the system one - `System.arraycopy(...)`

Reply:
just an idea: if check for null needed, can be like this - Optional.ofNullable(orig).map()